src/test/java/ParserTest.java

```diff
@@ -0,0 +1,20 @@
+import johnny.Parser;
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.fail;
+
+import java.util.ArrayList;
+
+public class ParserTest {
+
+    @Test
+    public void parse_addTodoTask_success() throws Exception{
```

Comment:
```suggestion
    public void parse_addTodoTask_success() throws Exception {
```
Missing whitespace between Exception and {